src/vs/workbench/contrib/terminal/common/terminalConfiguration.ts

```diff
@@ -398,6 +398,11 @@ export const terminalConfiguration: IConfigurationNode = {
 			type: 'boolean',
 			default: true
 		},
+		'terminal.integrated.mainSpawn': {
+			description: localize('terminal.integrated.mainSpwan', "Experimental: spawn remote terminals from the main process"),
+			type: 'boolean',
+			default: true
+		},
```

Comment:
We can remove this now, this will be the only way when https://github.com/microsoft/vscode/pull/116373 is merged as I'm pretty confident in it